### ISSUE TITLE
Move to ckan org

### ISSUE THREAD

**Issue by amercader** [pdelboca/ckan-extensions-monitor#4]
@pdelboca what do you think about:

1. Moving this repo to the ckan org, dropping ckanext-sitesearch
2. Creating fork in the okfn org that contains the non-official extensions that we use at okfn: sitesearch, saml2auth, versions, etc

We can document in the ckan org repo how users can set up their own monitor with their preferred extensions

**Comment by pdelboca** [comment]
## TL/DR

+1 on moving, but I would like to maintain only one with all the info we need to plan releases.

## More thoughts

Hey @amercader !

+1 on moving this to the CKAN Org.

However, I would like to keep only one repository and avoid duplicates. My reasons:
 - Workflows are not following best practices like suggested in #1 so forking means more maintenance work (even when is minimal work, it is still work)
 - I envisioned this repository also as a showcase of our extension ecosystem. A single place were one can see what's going on in the CKAN community. Something like https://github.com/ckan/awesome-ckan with the extra benefit of knowing if it's running on master or not.
 - Even when they are not maintained by CKAN, extensions like `sitesearch` and `saml2auth` are well known. Also, we use them as a reference and they impact our decision on releases. More repos means that we will need to pivot between them to have an informed decision.

### Middleground proposal:

What if we update the Readme to separate by vendor/maintainer? To clear point to the ones unde CKAN umbrella, under OKF umbrella and why not inviting other vendors to add theirs too. This could have also the benefit of providing a place for partners and vendors to "showcase" their portfolio.

**Comment by amercader** [comment]
@pdelboca thanks. I think that the work involved in maintaining one or separate repos is pretty similar, but in any case, my main concern in having one single repo with mixed "official" and other extensions might create the assumption that the tech team is responsible for all of them or that they all need to be green for a release to happen.
If we keep one repo let's clearly differentiate them in the README, using a separate table, adding a disclaimer or maintainer field, etc as you suggest
